### PR TITLE
FEATURE: SD-933 mark 7.5.0 deprecated, release 7.6.0

### PR DESCRIPTION
**Pull request by naibinh**
---

[Check PR owner responsibilities](https://confluence.intgdc.com/display/Development/Code-reviews#Code-reviews-Ownerresponsibilities)

Supported PR commands:

Command | Description
--- | ---
`extended test - examples` | Live examples tests
`extended test - storybook` | Storybook screenshot tests

See more [options](https://confluence.intgdc.com/display/kbhr/How+to+work+with+git+and+Github#HowtoworkwithgitandGithub-Extendedchecks).

# PR Checklist

- [ ] Verify code changes ([Checklist](https://confluence.intgdc.com/display/Development/Code-reviews+checklist), [Best practices](https://confluence.intgdc.com/display/Development/Code-reviews+best+practices))
- [ ] [Verify pull-request formalities](https://confluence.intgdc.com/display/Development/Code-reviews)
- [ ] Change was tested by using [gdc-dev-release](https://confluence.intgdc.com/display/~tomas.vojtasek/Private+NPM) in [gdc-analytical-designer](https://github.com/gooddata/gdc-analytical-designer) and [gdc-dashboards](https://github.com/gooddata/gdc-dashboards) (if applicable)
- [ ] Migration guide (for major changes) is mentioned in [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [ ] Successful `extended test - examples`
- [ ] Successful `extended test - storybook`
- [ ] Checked yarn.lock consistency (no dep. duplicities especially Goodstrap)

# Related PRs

- gdc-analytical-designer:
- gdc-dashboards:

# Related Jira tasks

### CODE REVIEW

**Comment by gdgate** [comment]
Build in pipeline check aborted.

File: CHANGELOG.md [hunk 1]
```diff
@@ -29,6 +29,12 @@ The REST API versions in the table are just for your information as the values a
 
 - Visualization rendering stuck due to an error that persisted after the error was gone
 
+## 7.5.0
+
+### Note
+
+- Deprecated version
```
Comment (no23reason) [inline CHANGELOG.md:36]: We should add some explanation for why is the version deprecated. Something like

> This version was released by accident, please use 7.6.0 instead.

Comment (naibinh) [reply CHANGELOG.md]: Updated.
Thank you

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7212/ : SUCCESS in 5m 41s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3210/ : SUCCESS in 3s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- gooddata-react-components-storybook-zuul-docker : SKIPPED
- gooddata-react-components-unit-tests-zuul-docker-el7 : SKIPPED
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1749/ : FAILURE in 11s
- gooddata-react-components-examples-zuul-docker : SKIPPED

**Comment by leosflodr** [comment]
approved by DC

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- gooddata-react-components-storybook-zuul-docker : SKIPPED
- gooddata-react-components-unit-tests-zuul-docker-el7 : SKIPPED
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1752/ : FAILURE in 12s
- gooddata-react-components-examples-zuul-docker : SKIPPED

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/3777/ : SUCCESS in 12m 26s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7217/ : SUCCESS in 3m 32s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1753/ : SUCCESS in 11s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/2575/ : FAILURE in 20m 44s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/3778/ : SUCCESS in 10m 28s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7218/ : SUCCESS in 2m 41s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1754/ : SUCCESS in 13s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/2577/ : FAILURE in 12m 48s

**Comment by naibinh** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-1574.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/3779/ : SUCCESS in 10m 45s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7219/ : SUCCESS in 7m 44s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1755/ : SUCCESS in 13s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/2579/ : FAILURE in 10m 54s

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3218/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/3156/ : FAILURE in 13m 05s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/3783/ : SUCCESS in 14m 07s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7222/ : SUCCESS in 3m 22s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1758/ : SUCCESS in 12s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/2583/ : SUCCESS in 9m 58s

**Pull request merged**